prisma-fmt/tests/text_document_completion/scenarios/referential_actions_in_progress_2/result.json

```diff
@@ -1,30 +1,10 @@
 {
   "isIncomplete": false,
   "items": [
-    {
-      "label": "Cascade",
-      "kind": 13,
-      "detail": "Delete the child records when the parent record is deleted."
-    },
     {
       "label": "Restrict",
       "kind": 13,
       "detail": "Prevent deleting a parent record as long as it is referenced."
-    },
-    {
-      "label": "NoAction",
-      "kind": 13,
-      "detail": "Prevent deleting a parent record as long as it is referenced."
-    },
-    {
-      "label": "SetNull",
-      "kind": 13,
-      "detail": "Set the referencing fields to NULL when the referenced record is deleted."
-    },
-    {
-      "label": "SetDefault",
-      "kind": 13,
-      "detail": "Set the referencing field's value to the default when the referenced record is deleted."
     }
```

Comment:
Can you elaborate on why these completions are gone?

Reply:
The positioning we have has been tightened. I explain it further [here](https://github.com/prisma/prisma-engines/pull/4934#discussion_r1655128473)

Reply:
Gotcha, this is great 👍